package/MDAnalysis/lib/picklable_file_io.py

```diff
@@ -39,6 +39,8 @@
 .. autofunction:: pickle_open
 
 """
+from __future__ import division, absolute_import
```

Comment:
Now that we're py3+ you shouldn't need future stuff anymore.

Reply:
Then I assume we should remove `no-absolute-import` from `pylintrc` in #2750  ?
```
************* Module MDAnalysis.lib.picklable_file_io
package/MDAnalysis/lib/picklable_file_io.py:42: [W1618(no-absolute-import), ] import missing `from __future__ import absolute_import`
```

Reply:
I think this is just happening because the PR fixing the pylint call hasn't been merged yet.

Reply:
Then the cleaner approach is to keep the legacy `from __future__` stuff in ongoing PRs and have a separate clean-up PR that gets rid of it everywhere together with the check instead of removing checks as part of one of many PRs that might be affected.

Reply:
Once the clean-up PR is merged, the ongoing PRs can merge/rebase and clean up themselves. At least that's my suggestion.

Reply:
The pylint PR was merged, if you merge develop you can start removing all Py 2 crud.

Reply:
@yuxuanzhuang please do remove this line, we have nearly removed all `__future__` imports now.